cmd/snap/cmd_run.go

```diff
@@ -1089,3 +1150,6 @@ func (x *cmdRun) runSnapConfine(info *snap.Info, securityTag, snapApp, hook stri
 		return syscallExec(cmd[0], cmd, envForExec(nil))
 	}
 }
+
+var cgroupCreateTransientScopeForTracking = cgroup.CreateTransientScopeForTracking
```

Comment:
We have the mocking, but not the actual tests to exercise the relevant branches and verify calls.

Reply:
As discussed on IRC I'm working on those now.